doc/reference/reference_lua/box_space/create_index.rst

```diff
@@ -361,50 +368,61 @@ Creating an index using the path option for map fields (JSON-path indexes)
 --------------------------------------------------------------------------------
 
 To create an index for a field that is a map (a path string and a scalar value),
-specify the path string during index_create, that is,
-:code:`parts={` :samp:`{field-number},'{data-type}',path = '{path-name}'` :code:`}`.
-The index type must be ``'tree'`` or ``'hash'`` and the field's contents
+specify the path string during index creation, like this:
+
+..  cssclass:: highlight
+..  parsed-literal::
+
+    :extsamp:`parts = {{*{field-number}*}, {*{'data-type'}*}, path = {*{'path-name'}*}}`
+
+The index type must be ``tree`` or ``hash`` and the contents of the field
 must always be maps with the same path.
 
 **Example 1 -- The simplest use of path:**
 
-..  code-block:: lua
+..  code-block:: tarantoolsession
 
-    -- Result will be - - [{'age': 44}]
-    box.schema.space.create('T')
-    box.space.T:create_index('I',{parts={{field = 1, type = 'scalar', path = 'age'}}})
-    box.space.T:insert{{age=44}}
-    box.space.T:select(44)
+    tarantool> box.schema.space.create('T')
+    tarantool> box.space.T:create_index('I',{parts = {{field = 1, type = 'scalar', path = 'age'}}})
+    tarantool> box.space.T:insert({{age = 44}})
+    tarantool> box.space.T:select(44)
+    ---
+    - [{'age': 44}]
 
 **Example 2 -- path plus format() plus JSON syntax to add clarity:**
 
 ..  code-block:: lua
 
-    -- Result will be: - [1, {'FIO': {'surname': 'Xi', 'firstname': 'Ahmed'}}]
-    s = box.schema.space.create('T')
-    format = {{'id', 'unsigned'}, {'data', 'map'}}
-    s:format(format)
-    parts = {{'data.FIO["firstname"]', 'str'}, {'data.FIO["surname"]', 'str'}}
-    i = s:create_index('info', {parts = parts})
-    s:insert({1, {FIO={firstname='Ahmed', surname='Xi'}}})
+    tarantool> my_space = box.schema.space.create('T')
+    tarantool> format = {{'id', 'unsigned'}, {'data', 'map'}}
+    tarantool> my_space:format(format)
+    tarantool> parts = {{'data.FIO["firstname"]', 'str'}, {'data.FIO["surname"]', 'str'}}
+    tarantool> my_index = my_space:create_index('info', {parts = parts})
+    tarantool> my_space:insert({1, {FIO = {firstname = 'Ahmed', surname = 'Xi'}}})
+    ---
+    - [1, {'FIO': {'surname': 'Xi', 'firstname': 'Ahmed'}}]
 
 **Note re storage engine:** vinyl supports only the TREE index type, and vinyl
 secondary indexes must be created before tuples are inserted.
 
 .. _box_space-path_multikey:
 
 --------------------------------------------------------------------------------
-Creating a multikey index using the path option with [*]
+Creating a multikey index using the path option with wildcard [*]
```

Comment:
```suggestion
Creating a multikey index using the path option with array index placeholder [*]
```

Later we explicitly say that `[*]` is called a placeholder

Reply:
`*` is a wildcard in some common sence. Indeed, here we have a very specific meaning for it. There's no need to add another term which we will never use again in this document.

Reply:
maybe it's better to omit it here at all? The heading becomes too long